.github/workflows/js-prs-issue.yml

```
@@ -14,8 +14,62 @@ jobs:
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     runs-on: ubuntu-latest
     steps:
+      - name: Get Alerts List
+        id: get-alerts-list
+        run: |
+          ALERTS_LIST=''
+
+          # Get list of supported modules
+          # Assumes CMS 5 is the most recent stable version
+          curl -s -o __modules.json https://raw.githubusercontent.com/silverstripe/supported-modules/5/modules.json
+          if ! jq -e '.' __modules.json >/dev/null 2>&1; then
```

```suggestion
          if ! [[ $(jq -e '.' __modules.json >/dev/null 2>&1) ]]; then
```
Make the if conditional consistent with all the other workflow files. This is untested, though it'll be something like this

---

This was directly copied from gha-ci:
https://github.com/silverstripe/gha-ci/blob/1/.github/workflows/ci.yml#L854-L858

I'm inclined to leave it as is since we know this works.

---

We discussed this offline - I'll update it in both places.

---

Done, and updated the links in the issue description for a re-run of this.

Note that I have ended up with `if ! [[ $(jq -e '.' __modules.json) ]] >/dev/null 2>&1; then`. We can't have the `>/dev/null 2>&1` part inside the brackets because then of course the `$()` has no output to capture.

---

This discussion is moved to https://github.com/silverstripe/.github/pull/124#discussion_r1344808107